js_modules/dagit/packages/core/src/ui/Filters/FilterDropdown.tsx

```diff
@@ -332,7 +339,7 @@ export const FilterDropdownButton = React.memo(({filters}: FilterDropdownButtonP
         canEscapeKeyClose
         popoverClassName="filter-dropdown"
         isOpen={isOpen}
-        position="bottom"
+        placement="bottom-start"
```

Comment:
Shifted the menu to be left-aligned flush with the button, instead of centered.

Reply:
oh sweet, thanks!